src/firefly/js/ui/ImageSelect.jsx

```diff
@@ -13,7 +13,7 @@ import FieldGroupUtils, {getFieldVal} from '../fieldGroup/FieldGroupUtils.js';
 import {dispatchComponentStateChange} from '../core/ComponentCntlr.js';
 import {updateSet} from '../util/WebUtil.js';
 
-import './ImageSelect.css';
+import './NewImageSearchPanel.css';
```

Comment:
Not duplicating the style is a good idea, but in this case, most of the style in ImageSelect.css is for ImageSelect only.  You should have just moved the relevant style from ImageSelect to NewImageSearchPanel.css.